selfdrive/car/toyota/interface.py

```diff
@@ -340,6 +340,10 @@ def update(self, c, can_strings):
         # while in standstill, send a user alert
         events.add(EventName.manualRestart)
 
+    if ret.vEgo < 0.001 and self.CS.pcm_acc_status == 7:
```

Comment:
doesn't the car show an alert? I don't like this alert creation delay hack

Reply:
No, the car only alerts when it reaches 180 seconds (and shift to p), in which case openpilot will panic as well for wrong gear.